src/native/libs/System.Globalization.Native/CMakeLists.txt

```
@@ -1,3 +1,6 @@
+# Required for StaticICULinking
+cmake_minimum_required(VERSION 3.20)
```

```suggestion
cmake_minimum_required(VERSION 3.16...3.20)
```

Can we change the minimum version to 3.16 to be consistent with other CMake scripts we ship to external customers?

---

What are the other CMake scripts that we ship?